source/platforms/bitbucket_server/BitBucketServerAPI.ts

```diff
@@ -248,6 +250,21 @@ export class BitBucketServerAPI {
 
     const url = `${this.repoCredentials.host}/${path}`
     this.d(`${method} ${url}`)
+
+    // Allow using a proxy configured through environmental variables
+    // Remember that to avoid the error "Error: self signed certificate in certificate chain"
+    // you should also do: "export NODE_TLS_REJECT_UNAUTHORIZED=0". See: https://github.com/request/request/issues/2061
+    let agent: Agent | undefined = undefined
+    let proxy: string | undefined = undefined
+    if (process.env.https_proxy) {
```

Comment:
I used those two env variables because they seem to be the standard for command line: https://askubuntu.com/questions/47379/how-to-use-a-proxy-on-the-command-line

What I realise now is that I should not add http and https by myself. Ill fix that

Reply:
👍 cool, makes sense - yep